spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -4233,6 +4234,27 @@ def __in_block_comment(cursor):
         cursor3.endEditBlock()
         return True
 
+    def create_new_cell(self):
+        firstline = '#%% Start Cell \r\n'
```

Comment:
```suggestion
        firstline = '#%% ' + self.get_line_separator()
```

* I like a lot the idea of enclosing selections between cell separators. But I don't think adding the extra `Star cell` is appropriate because the start of a cell is the end of the previous one. So that could introduce a bit of confusion for users because they could infer that the region before `Start cell` is not really a cell, when we know it is.
* This assumes that the line endings of all files are `\r\n`, but we can also handle `\n` (Linux) and `\r` (Mac). So, we need to use the method `get_line_separator` instead.